05-read-write-plot.Rmd

```diff
@@ -150,106 +253,22 @@ raster_filepath = system.file("raster/srtm.tif", package = "spDataLarge")
 single_layer = raster(raster_filepath)
 ```
 
-You can also read just one layer (band) from a multilayer file by specifying the `band` argument.
-
-```{r}
-raster_multilayer_filepath = system.file("raster/landsat.tif", package = "spDataLarge")
-single_layer_band3 = raster(raster_multilayer_filepath, band = 3)
-```
-
-The whole multilayer raster file is read by `brick()`.
-
-```{r}
-multilayer_layer_brick = brick(raster_multilayer_filepath)
-```
-
-Finally, many single and multilayer rasters could be read together using the `stack()` function:
```

Comment:
I deleted this part because we showed similar code in c2. But still there is a problem, in c2 we are already presenting how to read in raster data using raster, stack and brick. And then explain what the differences are. Hence, I would suggest to just show in c2 how to manually create a raster and how to stack two rasters while referring to c5 for a more detailed explanation regarding raster stacks and bricks (i.e. move great parts of the raster classes section to c5). What do you think?